cartography/intel/aws/eks.py

```diff
@@ -56,16 +59,42 @@ def load_eks_clusters(neo4j_session, cluster_data, region, current_aws_account_i
             ClusterArn=cluster['arn'],
             ClusterName=cluster['name'],
             ClusterEndpoint=cluster.get('endpoint'),
+            ClusterExposed=_process_access_policy(cluster),
+            ClusterRoleArn=cluster.get('roleArn'),
             ClusterVersion=cluster.get('version'),
             ClusterPlatformVersion=cluster.get('platformVersion'),
             ClusterStatus=cluster.get('status'),
             CreatedAt=str(cluster.get('createdAt')),
+            ClusterLogging=_process_logging(cluster),
             Region=region,
             aws_update_tag=aws_update_tag,
             AWS_ACCOUNT_ID=current_aws_account_id,
         )
 
 
+def _process_logging(cluster):
+    """
+    Parse cluster.logging.clusterLogging to verify if
+    at least one entry has audit logging set to Enabled.
+    """
+    logging = False
+    cluster_logging = cluster.get('logging', {}).get('clusterLogging')
+    if cluster_logging:
+        logging = any(filter(lambda x: 'audit' in x['types'] and x['enabled'], cluster_logging))
+    return logging
+
+
+def _process_access_policy(cluster):
```

Comment:
The pattern used for marking compute instances as internet-exposed is to have it as an [analysis job](https://github.com/lyft/cartography/blob/master/docs/dev/writing-analysis-jobs.md) here: https://github.com/lyft/cartography/tree/master/cartography/data/jobs/analysis

We might want to move it there for consistency.  Any thoughts @sachafaust?

Reply:
@marco-lancini I thought about it a bit more, and I think we should move this to an analysis job.  Let's put all derived fields in analysis jobs and have the python intel modules focus on the getting and setting.

Reply:
Seems reasonable. I would also add the endpointPublicAccess attribute on the cluster node also.

Reply:
Sounds good to me.

I've put it here because (after reviewing other intel modules like EC2), I had the impression that analysis jobs were used when there is the need to compute a value (e.g., for EC2 by checking security groups, etc.), whereas in this case the `internet_exposed` attribute is directly provided by the API (even if under another name)

Reply:
As a general rule of thumb, I think we should include API-provided attributes to the node in the python code (e.g. `endpointPublicAccess` or `s3.anonymous_actions`), and then set the `internet_exposed` flag only in analysis jobs.

Reply:
Gotcha! Will amend tomorrow :)

Reply:
Added an analysis job, let me know if it does sound right!